server/auth_function/lambda_function.py

```diff
@@ -29,10 +37,15 @@
     IDP_NAME_BCEID_BUSINESS: USER_TYPE_BCEID_BUSINESS,
     IDP_NAME_BCSC_DEV: USER_TYPE_BCSC_DEV,
     IDP_NAME_BCSC_TEST: USER_TYPE_BCSC_TEST,
-    IDP_NAME_BCSC_PROD: USER_TYPE_BCSC_PROD
+    IDP_NAME_BCSC_PROD: USER_TYPE_BCSC_PROD,
 }
 
 
+class AuditEventOutcome(str, Enum):
+    SUCCESS = 1
+    FAIL = 0
```

Comment:
Showing (1 or 0) on audit log could be hard to for audit person to understand (since this is for audit, not for debugging). 
(I understand this is copied from api-audit and it has the same; but initially I think I used "Success" and "Fail" to be more clear but I think api side was changed to 1/0).

Also if we like to use 1/0, then in Python, probably don't need (str) argument in the class (as initially previously the intention was for "string" enum, not the default enum (not 100% sure).

Reply:
I see... yeah, I just try to follow the api side, will wait before merge this one